### DISCUSSION TITLE
What are the best training parameters for training dreambooth on my face?

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#2945] [Q&A]
Can anyone share their best parameters that they used to train dreambooth on their face to create exactly same (realistic) pictures, using fast dreambooth notebook? And how many images are you using? What number of steps and learning rates you are using?

For me, I used about 14 images (512x512) and 1500 steps, 3e-6 unet learning rate, 350 text encoder training steps. But I didn't got good results with my training. 

As I am using colab free version, I need help from the community to tell me their settings and also, the example prompt they used to generate hyper realistic self images.

Help me out. Thanks

**Comment by unco-ctr** [comment]
It heavily depends on the model you're fine-tuning it for, quality of the pictures, uniformity of angles and prompts, there is no universal recipe. Your numbers look ok, try and practice on different models, prompts.
